apps/web/components/eventtype/EventLimitsTab.tsx

```diff
@@ -184,6 +196,7 @@ export const EventLimitsTab = ({ eventType }: Pick<EventTypeSetupProps, "eventTy
                     onChange={(val) => {
                       if (val) onChange(val.value);
                     }}
+                    isDisabled={shouldLockDisableProps("bookingLimits").disabled}
```

Comment:
This was removed recently to allow these fields to be unlocked for v1, now with v2 we need them as the team admin can discretely select their state